components/raftstore-v2/src/operation/command/write/ingest.rs

```diff
@@ -39,7 +39,7 @@ impl Store {
         ctx: &mut StoreContext<EK, ER, T>,
     ) -> Result<()> {
         let ssts = box_try!(ctx.sst_importer.list_ssts());
-        if ssts.is_empty() {
+        if ssts.is_empty() || ctx.sst_importer.regional_import_mode() {
```

Comment:
We should not block cleaning up SST of region A when region B is in import mode.

Reply:
1. region infos in sst_importer may not be update to date.
2. the regions may be created soon but now in this store now, but the ssts are uploaded in advance.
3. there's a background worker to check timeout and all regions will not be import mode eventually at some time.

Reply:
Then I think we need to use region's key range to check if the region should be in import mode or not. So it does not matter the region split/merge.